lib/chef/resource/yum_repository.rb

```diff
@@ -29,7 +29,7 @@ class YumRepository < Chef::Resource
       property :cost, String, regex: /^\d+$/
       property :clean_headers, [TrueClass, FalseClass], default: false # deprecated
       property :clean_metadata, [TrueClass, FalseClass], default: true
-      property :description, String, regex: /.*/, default: "Ye Ole Rpm Repo"
+      property :description, String, regex: /.*/, default: "Yum Repository"
```

Comment:
It seemed like a more professional default. The yum cookbook should probably get updated as well.